src/c20_server/flask_server.py

```diff
@@ -41,10 +41,12 @@ def _return_result():
 
 
 if __name__ == '__main__':
-    REDIS_SERVER = fakeredis.FakeServer()
-    REDIS_SERVER.connected = True
-    REDIS = fakeredis.FakeStrictRedis(server=REDIS_SERVER)
-    JOB_MANAGER = JobManager(database=REDIS)
-    JOB_MANAGER.add_job(DocumentsJob('1', 0, '12/28/19', '1/23/20'))
-    APP = create_app(JOB_MANAGER)
-    APP.run()
+    try:
+        redis.Redis().ping()
+    except redis.exceptions.ConnectionError:
+        print('Redis-server is not running!')
+    else:
+        JOB_MANAGER = JobManager(redis.Redis())
+        JOB_MANAGER.add_job(DocumentsJob('1', 0, '12/28/19', '1/23/20'))
+        APP = create_app(JOB_MANAGER)
+        APP.run()
```

Comment:
`main` is not tested.  It is part of the 5% of untested code.

But we have logic here that should be tested:  The server does not start if Redis isn't running.  This functionality should be factored out into a class that can be tested.

Reply:
let's discuss this further...